cli/main.rs

```diff
@@ -317,26 +317,28 @@ pub fn main() {
 
   let args: Vec<_> = env::args_os().collect();
   let current_exe_path = current_exe().unwrap();
-  let standalone =
-    standalone::extract_standalone(&current_exe_path, Cow::Borrowed(&args));
+  let maybe_standalone = match standalone::extract_standalone(
+    &current_exe_path,
+    Cow::Borrowed(&args),
+  ) {
+    Ok(standalone) => standalone,
+    // TODO(bartlomieju): doesn't handle exit code set by the runtime properly
```

Comment:
Looks like this todo can be removed actually?

Reply:
Sure, it appears it can be removed

Reply:
Oh wait, we should move this into standalone::run I think

Reply:
Yeah, it looks like this PR resolves this todo as well.